plugins/inputs/x509_cert/x509_cert.go

```diff
@@ -26,6 +25,14 @@ const sampleConfig = `
   ## Timeout for SSL connection
   # timeout = "5s"
 
+  ## Include Certificate Issuer information in tags
+  # include_issuer = false
+
+  ## Include Certificate SAN in tag
+  # include_san = false
+  ## Separator between each SAN in tag
+  # san_separator = ","
```

Comment:
Let's remove all of these options, the plugin user can use tagexclude/taginclude instead.  On the SAN separator we can just always use `,`, I think there is little reason to modify it but I suppose it could be done with a processor, and comma is consistent with our other uses.

Reply:
OK. I have removed them.